### PR TITLE
ZER-18 Test environment setup

### PR DESCRIPTION
**Pull request by ochupa**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -50,6 +54,10 @@ group :test do
   gem 'selenium-webdriver'
   # Easy installation and use of web drivers to run system tests with browsers
   gem 'webdrivers'
+  # Shoulda Matchers. Read more: https://github.com/thoughtbot/shoulda-matchers
+  gem 'shoulda-matchers', '~> 4.0'
+  # Database cleaner. Read more: https://github.com/DatabaseCleaner/database_cleaner
+  gem 'database_cleaner-active_record'
```
Comment (maksymst) [inline Gemfile:60]: I see you've installed this gem, but it should be configured in `spec_helper.rb`.
Read more 
https://github.com/DatabaseCleaner/database_cleaner#how-to-use
https://avdi.codes/configuring-database_cleaner-with-rails-rspec-capybara-and-selenium/

**Review by maksymst — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 1 above)
Comment (ochupa) [reply Gemfile:60]: Thanks. I configured database_cleaner according mentioned guide.

File: spec/database_cleaner.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+  config.before(:suite) do
+    DatabaseCleaner.clean_with(:truncation)
+  end
+
+  config.before(:each) do
+    DatabaseCleaner.strategy = :transaction
+  end
+
+  config.before(:each, :js => true) do
```
Comment (maksymst) [inline spec/database_cleaner.rb:11]: https://github.com/rubocop/ruby-style-guide#hash-literals

**Review by maksymst — CHANGES_REQUESTED** [review]

File: spec/database_cleaner.rb (hunk 2 above)
Comment (ochupa) [reply spec/database_cleaner.rb]: Thanks for your response! I fixed it.

**Review by maksymst — APPROVED** [review]

**Pull request merged**